setup.sh

```diff
@@ -0,0 +1,24 @@
+#! /usr/bin/env bash
+
+OLD_PWD=$(pwd)
+
+# Additional ZShell configs
```

Comment:
Not sure what this does, doesn't seem to be related to installation of the completion?